### PR TITLE
[PLAT-202] POWER/SQUARE/SQRT/CBRT functions implemented

### PR DESCRIPTION
**Pull request by Fly-Style**
Checklist:
- [x] Labels and Milestone set

### CODE REVIEW

**Comment by mmedenjak** [comment]
run-lab-run

**Comment by Fly-Style** [comment]
run-lab-run

File: hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java [hunk 1]
```diff
@@ -55,7 +56,8 @@
     @Parameterized.Parameters(name = "mode: {0}")
```
Comment (alex-dukhno) [inline hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java:56]:
```suggestion
    @Parameterized.Parameters(name = "function: {0}")
```

File: hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java [hunk 2]
```diff
@@ -178,19 +180,22 @@ public void testSerialization() {
 
     private static final class Mode {
 
-        private final String mode;
+        public final int mode;
+        public final String stringMode;
```
Comment (alex-dukhno) [inline hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java:184]: consider renaming fields `mode` to `functionType` (as reference to `type` field in `DoubleBiFunction`) and `stringMode` to `functionName`

File: hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java (hunk 1 above)
Comment (Fly-Style) [reply hazelcast-sql-core/src/test/java/com/hazelcast/sql/impl/expression/math/DoubleBiFunctionIntegrationTest.java:56]: I would not like to change the style given in `DoubleFunctionIntegrationTest`

**Review by viliam-durina — APPROVED** [review]
Entries need to be added to the refman, but I'm not sure where that is atm :)

**Review by viliam-durina — APPROVED** [review]

**Review by alex-dukhno — APPROVED** [review]

**Pull request merged**